pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -161,8 +161,17 @@ in rec {
       deps = [ ensureNewerSourcesForZipFilesHook ];
       substitutions = {
         inherit pythonInterpreter;
-    };
-  } ./venv-shell-hook.sh) {});
+      };
+    } ./venv-shell-hook.sh) {});
+
+  wheelDepsVersionsHook = callPackage ({ wheel }:
```

Comment:
What do you think about `wheelRelaxVersionsHook`. In my opinion it describes better what it does. Alternatively `pythonRelaxVersionsHook` since users not necessarily need to know about wheels. In that case, please prefix the variables also with `python` instead of `wheel`. Hopefully that makes it somewhat clear to users where these attributes are being used.

Reply:
Maybe:
- `wheelDepsHook` (removing the `Versions` since `remove` doesn't affect versions)
- `wheelDepsUtilsHook`
- `wheelFixDepsHook`

Not sure. Quite like the last one, but I know that the name `Fix` is kinda controversial in this context 😅 .

Reply:
Went with `pythonRelaxDeps` at the end. The reason is that `remove` is still a relaxation of the dependencies, and it made sense to be like this. However I also dropped the version from the name.

Hope this is a good name.